release_dashboard/lib/services/conductor.dart

```diff
@@ -9,6 +9,9 @@ import 'package:file/file.dart';
 ///
 /// This exists as a common interface for user interface to rely on.
 abstract class ConductorService {
+  /// Returns the directory where checkout is saved.
+  Directory get rootDirectory;
```

Comment:
this getter will be used to display the checkout directory in one of the substeps